### PR TITLE
Opengear ACM7008-2-L

### PR DESCRIPTION
**Pull request by cerealcable**
This is essentially exactly the same as the ACM7008-2-M but doesn't have SFP ports, only ethernet.

### CODE REVIEW

**Comment by cerealcable** [comment]
Commenting for GitHub to prevent this from staying stale.

**Comment by danner26** [comment]
Hey @cerealcable, please add a newline to the end of your file so it passes validation. Tag me once you update the PR so I can re-review! Thanks

Also, as a tip for the future, if you use pre-commit as described in the readme, it will fix these issues for you. If you have any questions feel free to let me know

File: device-types/Opengear/ACM7008-2-L.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+manufacturer: Opengear
+model: ACM7008-2-L
+slug: acm7008-2-l
+part_number: ACM7008-2-L
+u_height: 1
+is_full_depth: false
+comments: ACM7008 Console Server with 8 Console Ports on the rear of the device.
+console-server-ports:
```
Comment (danner26) [inline device-types/Opengear/ACM7008-2-L.yaml:9]: Is [this image](https://webobjects2.cdw.com/is/image/CDW/5651010d?$product-detail$) the right model? 

If so, it mgiht be better to lavel them exactly as they appear/show in the config

File: device-types/Opengear/ACM7008-2-L.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    type: rj-45
+  - name: Port 5
+    type: rj-45
+  - name: Port 6
+    type: rj-45
+  - name: Port 7
+    type: rj-45
+  - name: Port 8
+    type: rj-45
+power-ports:
```
Comment (danner26) [inline device-types/Opengear/ACM7008-2-L.yaml:26]: Is [this image](https://webobjects2.cdw.com/is/image/CDW/5651010d?$product-detail$) the right model? 

If so, please make sure the power type is correct. c14 is definitely not correct.

**Review by danner26 — CHANGES_REQUESTED** [review]

**Comment by danner26** [comment]
Hello @cerealcable, are you able to work on this?

**Comment by danner26** [comment]
Hello @cerealcable I am reaching back out again to see if you can work on this to get it past the finish line..
If this is open another 2 weeks it will end up going stale again, and we will have to close it until you or someone else can work on it 😞

**Pull request closed without merging**